modules/ROOT/pages/bolt/handshake.adoc

```diff
@@ -105,13 +105,13 @@ The range cannot span multiple major versions.
 
 .Example with versions 4.3 plus two previous minor versions, 4.2 and 4.1
 ----
-00 02 03 04
+01 02 03 04
```

Comment:
The first byte is reserved. It's recommended to send 0, but the server will likely ignore it either way.
```suggestion
00 02 03 04
```

Reply:
Thank you for pointing that out. Although I don't recall reading that specification in the documentation, I think it's not relevant. However, I believe including the clarification about the first reserved byte could be useful for other developers. What do you think?"

Reply:
The fact that the first byte is reserved is documented right above this example

https://github.com/neo4j/docs-bolt/blob/d4576b0032ddb6742a3f7708cc20092ede707e26/modules/ROOT/pages/bolt/handshake.adoc?plain=1#L98

The value `00` is not documented other than in the example. The reason being that setting it to anything but 0, will likely cause no harm for servers following the version handshake scheme as described here. However, if we decide to make user of that byte, all code setting the byte to something else will have to be double-checked to not conflict with the future semantics.

It's not a rule set in stone to set it to 0, but we'll probably assume that it's 0 for legacy clients when/if we design some semantics. That's just because all official drivers set it to 0 and all the examples in the docs show it being 0 as well.

Reply:
Thank you very much for the response, it is clear now